{{cookiecutter.project_slug}}/pyproject.toml

```
@@ -131,4 +136,9 @@ legacy_tox_ini = """
     ) %}
         py3{{python_version}}
     {%- endfor %}
+
+    [testenv:docs]
+    commands =
+        mkdocs build -s
```

I (recently) prefer putting full names for flags in CI so it's obvious as to what they do
```suggestion
        mkdocs build --strict
```